tests/test_instance.py

```diff
@@ -13,6 +14,34 @@
 from conftest import CREATE_INSTANCE_METHOD, DELETE_INSTANCE_METHOD, GET_INSTANCE_METHOD
 
 
+@pytest.fixture(scope="session", autouse=True, params=[True, False])
+def configuration_mock(request):
+    pytest.isSecure = request.param
+    # A valid configuration file setting up the uri and metadata
+    config_path = str("tests/config.json")
```

Comment:
Using static paths can lead multiple tests to interact with each other, you can use the pre-defined fixture `tmp_path` to get a unique place for this file. We do a similar thing in the test `test_initialize_from_configuration`.